docs/submit/install-ospool-ap.md

```diff
@@ -74,54 +76,69 @@ Initial Steps
 Be aware that hosting an access point comes with responsibilities, both for the administrators as
 well as end users of the system. The polices can be found in the [Acceptable Usage Policy document](ap-ospool-aup.md).
 
-### Register your access point in OSG Topology
-To be part of OSG, your access point should be registered with the OSG.
+### Register your Access Point in OSG Topology
+To be part of the OSPool, your AP must be registered in the OSG Topology system.
```

Comment:
```suggestion
To make use of OSPool capacity, your AP must be registered in the OSG Topology system.
```

In the NWO, the pool is a completely separate entity with its own ownership/policy. As pool owners, we are then allowed to decree that APs must do this and that to make use of our pool's capacity